fdbserver/workloads/MultiVersionClientControl.actor.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+ * DummyWorkload.actor.cpp
```

Comment:
Nit: You could rename this file to be something slightly a little different that the file in `fdbclient` just to improve the ease of use a little. Whatever name you choose, you'll want to update the name at this line.